examples/step-4/step-4.cc

```diff
@@ -253,7 +253,7 @@ template <int dim>
 void Step4<dim>::make_grid ()
```

Comment:
Please, 

    git checkout file

for all files, that are not in our scope. Step-4 are for sure out of scope.